unfetter-discover/unfetter-ui
# fixes unfetter-discover/unfetter#552

- updated webpack-dev-server to cut down on disconnects in latest chrome
- fixed horizontal scroll
- refresh graph on boundry change

## To Test
1. Visit a workproduct threat dashboard
2. Click the Fab + and add a new external report
3. Be sure to select new attackpatterns
4. save and close the modal
5. notice the graphs and kill chain table should update
